src/seedu/addressbook/ui/Formatter.java

```diff
@@ -34,6 +35,17 @@ public String format(String... messages) {
         return sb.toString();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
```

Comment:
What would be a better place to have this method?

Reply:
I think ReadOnlyPerson might be a better place to house this method